### ISSUE TITLE
(Windows, Accessibility): Screen reader does not announce tabs in the main program window

### ISSUE THREAD

**Issue by Futyn-Maker** [hiddify/hiddify-app#1527]
### Search first

- [X] I searched and no similar issues were found

### What Happened?

I am totally blind and use a computer using a screen reader (NVDA), a program that reads information or displays it in Braille. I found that while the program is generally quite accessible for screen readers, there is one major issue that negatively impacts the user experience. When navigating in the main program window using the Tab key, the screen reader announces elements that relate to the current tab, such as the quick settings button, the connection button, etc., but does not determine the tabs themselves, such as “home”, “configuration” ", "settings", "logs", "about" - instead, when I press the Tab key, I hear silence, although I can activate the corresponding element using Enter or Space keys. It would be great if the problem was fixed and the application became more accessible for blind users, this would expand the audience of this wonderful application even more!

### Minimal Reproducible Example (MRE)

1. Launch the screen reader on a computer with OS Windows (it is best to install NVDA, but the built-in Narrator is also suitable);
2. Open Hiddify;
3. Navigate through the controls using the Tab key, listen to what the screen reader says

### Expected Behavior

When the user navigates through the main menu items of the program with the elements “Home”, “Settings”, “Logs” etc. the screen reader must announce the names of these elements

### Version

2.5.7

### Platform/OS

Windows

### Additional Context

The screencast shows this problem, you need to watch it with soun

https://github.com/user-attachments/assets/70ce8f19-dae0-4489-ac96-94d67dfe2904

### Are you willing to submit a PR? If you know how to fix the bug.

- [ ] I'm willing to submit a PR (Thank you!)
